data/org.freedesktop.portal.WebExtensions.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2022 Canonical Ltd
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.WebExtensions:
+      @short_description: WebExtensions portal
+
+      The WebExtensions portal allows sandboxed web browsers to start
+      native messaging servers installed on the host system.
+
+      This documentation describes version 1 of this interface.
+  -->
+  <interface name="org.freedesktop.portal.WebExtensions">
+    <!--
+        CreateSession:
+        @options: Vardict with optional further information
+        @session_handle: Object path for the #org.freedesktop.portal.Session created by this call.
+
+        Create a native messaging session. A successfully created
+        session can at any time be closed using
+        org.freedesktop.portal.Session::Close, or may at any time be
+        closed by the portal implementation, which will be signalled
+        via org.freedesktop.portal.Session::Closed.
+
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>session_handle_token s</term>
+            <listitem><para>
+              A string that will be used as the last element of the session handle. Must be a valid
+              object path element. See the #org.freedesktop.portal.Session documentation for
+              more information about the session handle.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="CreateSession">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="session_handle" direction="out"/>
+    </method>
+    <!--
+        Start:
+        @session_handle: Object path for the #org.freedesktop.portal.Session objetc
```

Comment:
objetc typo

Reply:
@jhenstridge can you please fix the typo?

Reply:
@jhenstridge can you please fix the typo?